### PR TITLE
change tispark daily ci version

### PR DESCRIPTION
**Pull request by qidi1**
Signed-off-by: qidi1 <1083369179@qq.com>
 After this [pr](https://github.com/pingcap/tispark/pull/2492) merge, we need to change TiSpark Daily CI.

### CODE REVIEW

File: jenkins/pipelines/ci/tispark/daily.groovy [hunk 1]
```diff
@@ -205,22 +209,22 @@ def release_master() {
...
+                test_base("master", commitID, "tidb=$version pd=$version tiflash=$version tikv=$version profile=spark-3.1")
             },
 
             test3: {
-                test_base("master", commitID, "tidb=$version pd=$version tiflash=$version tikv=$version profile=spark3.2.1")
+                test_base("master", commitID, "tidb=$version pd=$version tiflash=$version tikv=$version profile=spark-3.2")
             },
     )
     // tiflash test
     parallel(
```
Comment (shiyuhang0) [inline jenkins/pipelines/ci/tispark/daily.groovy:220]: add spark-3.3

Comment (qidi1) [reply jenkins/pipelines/ci/tispark/daily.groovy:248]: already add

**Comment by shiyuhang0** [comment]
LGTM

**Review by purelind — APPROVED** [review]
LGTM

**Pull request merged**